source/solvers/navier_stokes_scratch_data.cc

```diff
@@ -102,10 +102,12 @@ NavierStokesScratchData<dim>::enable_vof(const FiniteElement<dim> &fe,
   phase_gradient_values = std::vector<Tensor<1, dim>>(this->n_q_points);
 
   // Allocate physical properties
-  density_0   = std::vector<double>(n_q_points);
-  density_1   = std::vector<double>(n_q_points);
-  viscosity_0 = std::vector<double>(n_q_points);
-  viscosity_1 = std::vector<double>(n_q_points);
+  density_0           = std::vector<double>(n_q_points);
+  density_1           = std::vector<double>(n_q_points);
+  viscosity_0         = std::vector<double>(n_q_points);
+  viscosity_1         = std::vector<double>(n_q_points);
+  thermal_expansion_0 = std::vector<double>(n_q_points);
+  thermal_expansion_1 = std::vector<double>(n_q_points);
```

Comment:
it's not used in this PR, right? It will be in a next PR?

Reply:
It is used right now...